src/librustdoc/doctest/make.rs

```diff
@@ -210,53 +277,53 @@ impl DocTestBuilder {
     /// lines before the test code begins.
     pub(crate) fn generate_unique_doctest(
         &self,
-        test_code: &str,
+        doctest_code: &str,
```

Comment:
Nit: in `extracted.rs`, we use the name `doctest_code` to refer to the fully processed code, while here we are using it to refer to the unprocessed source.  this seems needlessly confusing.